src/framework/visualization/generateVizTemplate.ts

```diff
@@ -0,0 +1,35 @@
+import { VizNode, VizEdge, VizOptions } from './types/viz';
+
+/**
+ * Creates the html for the viz template
+ */
+export function generateVizTemplate(
```

Comment:
Is this generating template, or is it generating the actual content? Perhaps `generateVizHTML` or `generateVizDocument`?

Reply:
Good point, it does generate the HTML. I'll update that.